### PR TITLE
Add support for Jessie

### PR DESCRIPTION
**Pull request by cdonati**
Goes with https://github.com/AppScale/appscale-tools/pull/671

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4315/

File: AppController/lib/ejabberd.rb [hunk 1]
```diff
@@ -110,6 +110,19 @@ def self.get_ejabberd_version
     major_version
   end
 
+  def self.update_ctl_config
+    config_path = '/etc/ejabberd/ejabberdctl.cfg'
```
Comment (obino) [inline AppController/lib/ejabberd.rb]: Should this be a constant at the beginning of the file?

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4329/

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/4393/

**Pull request merged**